tests/Functional/TerminusTestBase.php

```diff
@@ -183,7 +183,7 @@ protected function assertTerminusCommandResultEqualsInAttempts(
         callable $callable,
         $expected,
         int $attempts = 24,
-        int $intervalSeconds = 10
+        int $intervalSeconds = 30
```

Comment:
So I guess this is our version of the backoff-and-retry logic our customers frequently need to incorporate into their CI processes?  We know terminus commands frequently fail due to stability issues in the backend API, so our own test code is set up to wait and retry many times until it succeeds?

We recently made changes to terminus's internal backoff-and-retry logic.  If I understand how things work correctly, terminus's backoff-and-retry logic will run its full course in every `$attempt` thats made here.  If terminus's internal logic is so inadequate that we need to run it 24x with 30 second delays between each, it implies to me that the internal backoff-and-retry logic needs more adjustment.